src/StaticAnalysis.ruleset

```diff
@@ -10,7 +10,6 @@
     <!-- RTTI is off because Windows OS policies believe RTTI has too much binary size impact for the value and is less portable than RTTI-off modules. -->
     <Rule Id="C26466" Action="None" />
     <!-- This one has caught us off guard as it suddenly showed up. Re-enablement is going to be in #2941 -->

```

Comment:
should probably get rid of the comment above it too
```suggestion
```

Reply:
Oh man this is a mistake. This is just for my testing.   It isn’t ready yet. Will restore this later.